ocs_ci/deployment/deployment.py

```diff
@@ -2933,177 +2979,40 @@ def deploy(self):
-        update bucket name and s3 storage link
-
-        """
-        oadp_data = templating.load_yaml(constants.ACM_DPA)
-        oadp_data["spec"]["backupLocations"][0]["velero"]["objectStorage"][
-            "bucket"
-        ] = self.meta_obj.bucket_name
-        oadp_yaml = tempfile.NamedTemporaryFile(mode="w+", prefix="oadp", delete=False)
-        templating.dump_data_to_temp_yaml(oadp_data, oadp_yaml.name)
-        run_cmd(f"oc create -f {oadp_yaml.name}")
-        # Validation
-        self.validate_dpa()
-
-    @retry((CommandFailed, MDRDeploymentException), tries=10, delay=10)
-    def validate_dpa(self):
-        """
-        Validate
-        1. 3 restic pods
-        2. 1 velero pod
-        3. backupstoragelocation resource in "Available" phase
-
-        """
-        # Check restic pods.
-        # Restic pods have been renamed to node-agent after oadp 1.2
-        oadp_version = get_oadp_version()
-
-        if version.compare_versions(f"{oadp_version} >= 1.2"):
-            restic_pod_prefix = "node-agent"
+        if version.compare_versions(f"{acm_version} >= 2.10"):
+            logger.info("Skipping Enabling Managed ServiceAccount")
         else:
-            restic_pod_prefix = "restic"
-        restic_list = get_pods_having_label(
-            f"name={restic_pod_prefix}", constants.ACM_HUB_BACKUP_NAMESPACE
-        )
-        if len(restic_list) != constants.MDR_RESTIC_POD_COUNT:
-            raise MDRDeploymentException("restic pod count mismatch")
-        for pod in restic_list:
-            if pod["status"]["phase"] != "Running":
-                raise MDRDeploymentException("restic pod not in 'Running' phase")
-
-        # Check velero pod
-        veleropod = get_pods_having_label(
-            "app.kubernetes.io/name=velero", constants.ACM_HUB_BACKUP_NAMESPACE
-        )
-        if len(veleropod) != constants.MDR_VELERO_POD_COUNT:
-            raise MDRDeploymentException("Velero pod count mismatch")
-        if veleropod[0]["status"]["phase"] != "Running":
-            raise MDRDeploymentException("Velero pod not in 'Running' phase")
-
-        # Check backupstoragelocation resource in "Available" phase
-        backupstorage = ocp.OCP(
-            kind="BackupStorageLocation",
-            resource_name="default",
-            namespace=constants.ACM_HUB_BACKUP_NAMESPACE,
-        )
-        resource = backupstorage.get()
-        if resource["status"].get("phase") != "Available":
-            raise MDRDeploymentException(
-                "Backupstoragelocation resource is no in 'Avaialble' phase"
-            )
-        logger.info("Dataprotection application successful")
+            enable_managed_serviceaccount()
 
-    def create_generic_credentials(self):
-        s3_cred_str = (
-            "[default]\n"
-            f"aws_access_key_id={self.meta_obj.access_key}\n"
-            f"aws_secret_access_key={self.meta_obj.secret_key}\n"
+        validate_policy_compliance_status(
```

Comment:
The policy would be complaint on active when all the backups are created and on passive when all the backups are created matching that of active. So this check should be done post creating `backschedule.yaml` resource on the active hub with proper timeouts and re-tries.

Reply:
i will remove this verification step from here